README.md

```diff
@@ -1,3 +1,9 @@
 #Atom Workshop
 
-A single tool for all atom types.
\ No newline at end of file
+A single tool for all atom types.
+
+## Running locally
```

Comment:
If you wanted to be really explicit you could mention that you need aws cli installed before running the script